docs/TimelineGuide.md

```diff
@@ -167,20 +182,32 @@ which is still excellent.
 These are guidelines that cactbot tries to follow for timelines.
 
 * add syncs for everything possible
-* always add an Engage! entry, but add syncs in case there's no /countdown
 * if the first boss action is an auto-attack, consider adding a sync for
-  the first auto-attack or at least the first "starts using" line
-* include the command line used to generate the timeline in a comment at the top
-* prefer actions for syncs over rp text, but rp text syncs if that's the only option
-* if you do sync a phase with rp text, add a large window sync for an action
-* use original names for ability text as much as possible
-* loops should use `jump` instead of having previous abilities have large windows
+  the first auto-attack, or at least the first "starts using" line.
```

Comment:
I think we should change this to say start on combat if there's only one boss fight in the zone.

Reply:
Maybe something along the lines of:
```
If there is only one boss in a zone, the timeline should start on entering combat.
(The timeline utility should handle this automatically.) Unless there's a pressing reason not to, 
add a wide sync to the first usage of a skill that isn't an auto-attack. 
This ensures that even if combat entry is early or late, the timeline will still be synced correctly.
```

Reply:
I think combat entry should be pretty reliable? Certainly more reliable than auto attacks. I guess my thinking is that with auto attacks, you'd want a wide sync, but ideally we shouldn't need to start from autos any more. The other reason for a wide sync is if you are starting from a combat log line (zone seal?) and you're worried people might not have their combat log turned on.

Reply:
I guess I'm mostly thinking of situations where the boss is pulled but the first auto timing, and therefore the entire timeline, is different depending on whether the pulling tank runs around or not. I've made an update that should probably be close to what you're thinking?

Reply:
Yeah, this is what I was looking for! And, re: tank running around, if you don't sync on autos, is this still an issue? Like, if the boss casts a raidwide at t=11, won't it still do that regardless of how many autos occurred? In other words, even if no autos, my understanding is that "modern" fights are still on script and won't adjust. I'm thinking of the video where people were pulling p5s very carefully with provokes and running and rescue and shirk to avoid any autos until the raidwide so that fflogs didn't start the fight until then and so two minutes didn't line up in awkward places and they'd still get a fast kill time.

Reply:
I don't know. I haven't specifically checked whether it was changed recently. I know there were some timelines since Endwalker started where I noticed some slippage in the time to the first raidwide, but presumably running multiple times and verifying should catch any such slip. Regardless, the updated wording should cover it.